index.html

```diff
@@ -350,58 +331,137 @@ <h2>With COSE</h2>
-    associated with a <a data-cite="VC-DATA-MODEL#dfn-holders">holder</a>, be protected at specific 
-    assurance levels.
-    (See <a href="https://pages.nist.gov/800-63-3-Implementation-Resources/63B/AAL/">NIST 800-63-3: Authenticator Assurance Levels</a>.)
-    </p>
-    <p class="note">
-      Also see <a href="https://datatracker.ietf.org/doc/draft-looker-oauth-attestation-based-client-auth/">
-        OAuth 2.0 Attestation-Based Client Authentication
-      </a>.
-    </p>
-  </section> 
+
+    <section class="normative">
+      <h2>JOSE Header Parameters and JWT Claims</h2>
+
+      <p>
+      When present in
+      the <a data-cite="RFC7515#section-4">JOSE Header</a> or
+      the <a data-cite="RFC7519#section-4">JWT Claims Set</a>,
+      members registered in
+      the IANA <a href="https://www.iana.org/assignments/jwt/jwt.xhtml">JSON Web Token Claims</a> registry or
+      the IANA <a href="https://www.iana.org/assignments/jose/jose.xhtml">JSON Web Signature and Encryption Header Parameters</a> registry
+      are to be interpreted as defined by the specifications referenced in the registries.
+      </p>
+      <p>
+        The normative statements in
+	<a data-cite="RFC7515#section-4.1">Registered Header Parameter Names</a>,
+        <a data-cite="RFC7519#section-5">JOSE Header</a>, and
+	<a data-cite="RFC7519#section-5.3">Replicating Claims as Header Parameters</a>
+        apply to securing credentials and presentations.
+      </p>
+      <p>
+        The unencoded JOSE Header is JSON
+        (`application/json`), not JSON-LD (`application/ld+json`).
+      </p>
+      <p>
+        It is RECOMMENDED to use
+	the IANA <a href="https://www.iana.org/assignments/jwt/jwt.xhtml">JSON Web Token Claims</a> registry and
+	the IANA <a href="https://www.iana.org/assignments/jose/jose.xhtml">JSON Web Signature and Encryption Header Parameters</a> registry
+        to identify any claims and header parameters that might be confused with
+        members defined by [[VC-DATA-MODEL-2.0]]. These include but are not
+        limited to: <code>iss</code>, <code>kid</code>,
+        <code>alg</code>, <code>iat</code>,
+        <code>exp</code>, and <code>cnf</code>.
+      </p>
+      <p>
+        When the <code>iat</code> and/or <code>exp</code> JWT claims are present,
+        they represent the issuance and expiration time of the signature, respectively.
+        Note that these are different from the <code>validFrom</code> and <code>validUntil</code> properties
+        defined in <a data-cite="VC-DATA-MODEL-2.0#validity-period">Validity Period</a>,
+        which represent the validity of the data that is being secured.
+      </p>
+      <p>
+        The JWT Claim Names <code>vc</code> and <code>vp</code>
+        MUST NOT be present.
+      </p>
+      <p>
+        Additional members may be present as header parameters and claims.
+	If they are not understood, they MUST be ignored.
```

Comment:
I wonder if it's worth further defining how claims can be understood....(like in the case of a context or json schema)

Reply:
This is a standard clause enabling extensibility.  Extensions are understood when implementations process the claims defined by those extensions.  But I don't know that expanding on this point at this particular location would add much value.  I'm prone to keep with the normal, simple formulation.

Reply:
we should definitely not be talking about understanding claims in this document.

that is for the core data model, or profiles.

Reply:
This clause was already present in the specification as:
> Additional members may be present. If they are not understood, they MUST be ignored.

Per my earlier comment to Gabe, this is a standard clause enabling extensibility.  I'm very reluctant to remove it.  Not having it opens the door to interpretations in which anything unrecognized results in an error - making extensibility impossible.